CONTRIBUTING.md

```diff
@@ -55,6 +55,9 @@ properties:
 - `url`: same as the [`url`](README.md#url) property in `index.json`.
 - `shortname`: same as the [`shortname`](README.md#shortname) property in
 `index.json`.
+- `forkOf`: same as the [`forkOf`](README.md#forkof) property in `index.json`.
+No need to set `seriesComposition` to `"fork"` when this property is set, the
+build logic will take care of that automatically.
```

Comment:
maybe that should be mentioned explicitly in the `seriesComposition` entry (more specifically, IIUC, it is forbidden to set `fork` explicitly)

Reply:
I completed the entry. It wasn't forbidden to set `fork` explicitly, actually. Now it is ;)